src/signals/incident-management/containers/IncidentSplitContainer/index.js

```diff
@@ -16,62 +18,76 @@ import { splitIncident } from './actions';
 
 import reducer from './reducer';
 import saga from './saga';
-import './style.scss';
 
 import SplitDetail from './components/SplitDetail';
 import SplitForm from './components/SplitForm';
 
-export class IncidentSplitContainer extends React.Component { // eslint-disable-line react/prefer-stateless-function
-  constructor(props) {
-    super(props);
-
-    this.handleSubmit = this.handleSubmit.bind(this);
-    this.handleCancel = this.handleCancel.bind(this);
-  }
-
-  componentDidMount() {
-    this.props.onRequestIncident(this.props.id);
-    this.props.onRequestAttachments(this.props.id);
-  }
-
-  handleSubmit(splitForm) {
-    this.props.onSplitIncident(splitForm);
-  }
-
-  handleCancel() {
-    this.props.onGoBack();
-  }
-
-  render() {
-    const { categories } = this.props;
-    const { incident, attachments, loading, stadsdeelList, priorityList } = this.props.incidentModel;
-    return (
-      <div className="incident-split-container">
+const StyledH1 = styled(Heading)`
+  font-weight: normal;
+  margin-bottom: 30px;
+`;
+
+const StyledWrapper = styled.div`
+  min-height: 800px;
+`;
+
+export const IncidentSplitContainer = ({
+  id,
+  incidentModel: {
+    incident,
+    attachments,
+    loading,
+    stadsdeelList,
+    priorityList,
+  },
+  categories,
+  onRequestIncident,
+  onRequestAttachments,
+  onSplitIncident,
+  onGoBack,
+ }) => { // eslint-disable-line react/prefer-stateless-function
+  useEffect(() => {
+    onRequestIncident(id);
+    onRequestAttachments(id);
+  }, []);
+
+  return (
+    <StyledWrapper>
+      <Row>
         {loading ? <LoadingIndicator /> :
-        (
-          <div className="row">
-            <div className="col-8">
-              <SplitForm
-                incident={incident}
-                attachments={attachments}
-                subcategories={categories.sub}
-                priorityList={priorityList}
-                handleSubmit={this.handleSubmit}
-                handleCancel={this.handleCancel}
-              />
-            </div>
-            <div className="col-4">
-              <SplitDetail
-                incident={incident}
-                stadsdeelList={stadsdeelList}
-              />
-            </div>
-          </div>
-        )}
-      </div>
-    );
-  }
-}
+        <Fragment>
+          <Column span={12}>
+            <StyledH1 $as="h1">Splitsen</StyledH1>
+          </Column>
+
+          <Column
+            span={7}
```

Comment:
Why not set spans for narrower screens?

Reply:
Herjen wants a difficult order for mobile so we will put this in a new user story